mu-plugins/osi-api/osi-api.php

```diff
@@ -49,17 +49,17 @@ public function register_routes() {
 				'callback'            => array( $this, 'get_licenses' ),
 				'permission_callback' => '__return_true',
 				'args'                => array(
-					'name'    => array(
+					'license_name' => array(
```

Comment:
@gin0115 would it be possible to use `id` here? If not, `license_name` is fine.

Reply:
we can do, but this is a hidden url param that we dont call. Its only really used on the internal side of things

Reply:
you will still call opensource.org/api/licenses?name=mit

Reply:
this is now up on staging
https://open-source-initiative-development.mystagingwebsite.com/api/licenses?id=mit

just adding to this pr